lib/utils.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Parse command line arguments
+ */
+function argv (name, type, defaultValue = null) {
+  const i = process.argv.indexOf('--' + name)
+  if (type === Boolean) return i > -1
+  if (i === -1) return defaultValue
+
+  const hasValue = i < process.argv.length - 1
+  if (!hasValue) return defaultValue
+
+  let value = process.argv[i + 1]
```

Comment:
would rename this function to `getArgValue` or something similar so it's more intuitive what is should do - especially since it's pure js code :D 

Don't understand the `Boolean` check - you assume if it's a boolean flag that if it's set it's always true? 
`--isSomething false` would return true for `argv("isSomething", Boolean)

If the flag is set I'd first always fetch the value and interpret it later for the specific type